src/classes/ADDR_CopyAddrHHObjBTN_TEST.cls

```diff
@@ -47,7 +47,7 @@ private with sharing class ADDR_CopyAddrHHObjBTN_TEST {
     @TestSetup
     private static void createRecordData() {
         npe01__Contacts_and_Orgs_Settings__c contactSettingsForTests = UTIL_CustomSettingsFacade.getContactsSettingsForTests(
-            new npe01__Contacts_and_Orgs_Settings__c (npe01__Account_Processor__c = CAO_Constants.ONE_TO_ONE_PROCESSOR));
+            new npe01__Contacts_and_Orgs_Settings__c (npe01__Account_Processor__c = CAO_Constants.HH_ACCOUNT_PROCESSOR));
```

Comment:
What is the reason to change pre-existing condition?

Reply:
As I mentioned earlier, I had issues in verifying the creation of address record when an account has address in test. In order for the address record to be created, it has to be HH account, that's the reason I changed. I verified all other tests are passing as well.